client/src/routes/home/PageNotClaimed.jsx

```diff
@@ -0,0 +1,73 @@
+import React, { useState } from 'react'
```

Comment:
please rename PageNotClaimed to DomainNotClaimed

Reply:
The component was renamed to DomainNotClaimed